From 99a228cd145ca0e63b7067848de88ba902d5f502 Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Wed, 31 Jul 2013 08:03:48 +0200
Subject: [PATCH 041/212] pc: Drop RHEL-6 compat_props virtio-serial-pci.{max_ports, vectors}

Message-id: <1375201922-6794-7-git-send-email-armbru@redhat.com>
Patchwork-id: 52819
O-Subject: [RHEL-7 PATCH v3 06/15] pc: Drop RHEL-6 compat_props virtio-serial-pci.{max_ports, vectors}
Bugzilla: 983991
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Bandan Das <bsd@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>

RHEL-6 has them only in RHEL-5 machine types.  See qemu-kvm-rhel6
commit aad5614.

Signed-off-by: Markus Armbruster <armbru@redhat.com>
---
 hw/i386/pc_piix.c |    8 --------
 1 files changed, 0 insertions(+), 8 deletions(-)

diff --git a/hw/i386/pc_piix.c b/hw/i386/pc_piix.c
index 5c1fe1e..00170cf 100644
--- a/hw/i386/pc_piix.c
+++ b/hw/i386/pc_piix.c
@@ -1092,14 +1092,6 @@ static QEMUMachine pc_machine_rhel700 = {
         .property = "use_broken_id",\
         .value    = stringify(1),\
     },{\
-        .driver   = "virtio-serial-pci",\
-        .property = "max_ports",\
-        .value    = stringify(1),\
-    },{\
-        .driver   = "virtio-serial-pci",\
-        .property = "vectors",\
-        .value    = stringify(0),\
-    },{\
         .driver   = "VGA",\
         .property = "rombar",\
         .value    = stringify(0),\
-- 
1.7.1

